File: Modul 4/PRAK403_2410817310016_ARIFFATAHILLAHPERMANA.c
#include <stdio.h>

int main() {
    int angka1, angka2;

    scanf("%d %d", &angka1,&angka2);
    if (angka1 < angka2) {
        for (int i = angka1; i <= angka2; ++i) {
            printf("%d %d", i, angka1 + angka2 - i);
            if (i < angka2) {
                printf(" - ");
            }
        }
    } else {
        for (int i = angka1; i >= angka2; --i) {
            printf("%d %d", i, angka1 + angka2 - i);
            if (i > angka2) {
                printf(" - ");
            }
        }
    }
    printf("\n");

    return 0;
}

File: Modul 4/PRAK403_2410817310016_ARIFFATAHILLAHPERMANA.py
angka1, angka2 = map(int, input("").split())
if angka1 < angka2:
    for i in range(angka1, angka2 + 1):
        print(i, angka1 + angka2 - i, end="")
        if i < angka2:
            print(" - ", end="")
else:
    for i in range(angka1, angka2 - 1, -1):
        print(i, angka1 + angka2 - i, end="")
        if i > angka2:
            print(" - ", end="")
print()